tests/bluetooth/host/id/bt_id_create/src/test_suite_invalid_inputs.c

```diff
@@ -78,26 +78,6 @@ ZTEST(bt_id_create_invalid_inputs, test_id_list_is_full)
 	zassert_true(err == -ENOMEM, "Unexpected error code '%d' was returned", err);
 }
 
-/*
- *  Test invalid input arguments to bt_id_create() by using a valid address of type public and using
- *  NULL value for the IRK.
- *
- *  Constraints:
- *   - A valid address of type public is used
- *   - Input IRK is NULL
- *
- *  Expected behaviour:
- *   - '-EINVAL' error code is returned representing invalid values were used.
- */
-ZTEST(bt_id_create_invalid_inputs, test_public_address)
```

Comment:
this behavior is still correct if `CONFIG_BT_HCI_SET_PUBLIC_ADDR=n`.
Please don't remove the test.

Reply:
Added a `ztest_tesst_skip()` instead.